modules/service/src/main/scala/lucuma/odb/api/service/Init.scala

```diff
@@ -45,7 +49,7 @@ object Init {
     "point": {
       "bandNormalized": {
         "sed": {
-          "galaxy": "SPIRAL"
+          "stellarLibrary": "O5_V"
```

Comment:
This should really be a `"galaxy": "SPIRAL"` as it was before, but the ITC isn't currently accepting it and I wanted at least one observation to be defined sufficiently to generate ITC results.